packages/internal/test/types.test-d.ts

```diff
@@ -16,16 +17,16 @@ import type { StorageNode } from '../src/lib-chainStorage.js';
       realCount: 0,
     },
   };
-  expectType<{ m1: () => void }>(attenuate(obj, { m1: true }));
-  expectType<{ m2: () => void }>(attenuate(obj, { m2: 'pick' }));
+  expectType<{ m1?: () => void }>(attenuateOne(obj, ['m1']));
+  expectType<{ m2?: () => void }>(attenuateOne(obj, ['m2']));
```

Comment:
This seems to point out that `attenuateOne` is a step backwards from `deepPick`, but we should at least preserve existing expectations of the latter.
```suggestion

  expectType<{ m1?: () => void }>(attenuateOne(obj, ['m1']));
  expectType<{ m2?: () => void }>(attenuateOne(obj, ['m2']));

  expectType<{ m1: () => void }>(deepPick(obj, { m1: true }));
  expectType<{ m2: () => void }>(deepPick(obj, { m2: 'pick' }));
```